### ISSUE TITLE
core 3.1.2 tests does not have correct schema for aggregated status.

### ISSUE THREAD

**Issue by sveitech** [rsmp-nordic/rsmp_validator#444]
We are getting this schema error when testing core 3.1.2:

```
   2024-09-11T13:01:58.240Z  54056  AA+BBCCC=DDD         AA+BBCCC=DDDEEFFF    08ed  Received invalid AggregatedStatus schema errors: /se/0, boolean, , /se/1, boolean, , /se/2, boolean, , /se/3, boolean, , /se/4, boolean, , /se/5, boolean, , /se/6, boolean, , /se/7, boolean,  {
    "mType": "rSMsg",
    "type": "AggregatedStatus",
    "mId": "08edbdab-7983-4cb2-8353-d0cd0b03e0c6",
    "ntsOId": "AA+BBCCC=DDDEEFFF",
    "xNId": "",
    "cId": "AA+BBCCC=DDDEEFFF",
    "aSTS": "2024-09-11T13:01:58.230Z",
    "se": [
        "false",
        "false",
        "false",
        "false",
        "true",
        "true",
        "false",
        "false"
    ],
    "fP": null,
    "fS": null
}
    2024-09-11T13:01:58.240Z  54056  AA+BBCCC=DDD         -                    -     Sent MessageNotAck for AggregatedStatus 08ed  {"mType":"rSMsg","type":"MessageNotAck","rea":"schema errors: /se/0, boolean, , /se/1, boolean, , /se/2, boolean, , /se/3, boolean, , /se/4, boolean, , /se/5, boolean, , /se/6, boolean, , /se/7, boolean,","oMId":"08edbdab-7983-4cb2-8353-d0cd0b03e0c6"}
```

For core 3.1.2, the "se" array are strings, not booleans. See the spec https://rsmp-nordic.github.io/rsmp_core/3.1.2/applicability/usage_of_json.html#aggregated-status-message

**Comment by emiltin** [comment]
Yes, the 3.1.2 JSON section show the bits as  the strings "true" and "false", from 3.1.3 they are booleans. But the section describing AggregatedStatus did not change. So the question si whether booleans has been used always, and it was just the JSNO example that was wrong. It seems simpler to use booleans in all core versions.

@otterdahl input?

**Comment by sveitech** [comment]
We have had our RSMP daemon connected to 3.1.2 servers back in the day, when 3.1.2 was "new" :). They were using strings for the values.

**Comment by otterdahl** [comment]
Correct. The examples in 3.1.2 used string for aggregated status. In RSMP 3.1.3 this changed to booleans.

Also:

Version 3.1.2:
`Please note that the JSON elements are formatted as JSON string elements and not JSON number or JSON boolean.`

Version 3.1.3:
`Please note that the JSON elements are formatted as JSON string elements and not as JSON number or as JSON boolean elements, with the exception of the message type “aggregated status” where JSON boolean elements are used.`

However, I think the RSMP simulator has always used booleans for aggregated status. Existing implementations may expect booleans.

**Comment by sveitech** [comment]
I had a look in our code-base. We have been using string versions as the elements in aggregated status for core 3.1.2 always (since 2014). I think we would have noticed if any 3.1.2 servers expected real JSON booleans.

**Comment by sveitech** [comment]
![image](https://github.com/user-attachments/assets/d2745494-47fc-4103-83e1-783275b8063c)

This is the official 3.1.2 core document we received back in 2012. This document also shows it as strings:

![image](https://github.com/user-attachments/assets/96e4fca1-d0bf-4488-979b-abfeaa8cd423)

**Comment by emiltin** [comment]
OK I'll change the schema to use string for the "se" array items for core 3.1.2.

**Comment by emiltin** [comment]
transfering to the schema repo at https://github.com/rsmp-nordic/rsmp_schema/issues
